discovery.md

```diff
@@ -0,0 +1,16 @@
+# Box-client discovery: two options
+
+## Option 1: --no-cloud
```

Comment:
So why is this a build-time option? Why not something that can be configured locally (with a big red warning somewhere, of course).